manifests/integrations/activemq_xml.pp

```diff
@@ -63,7 +63,7 @@
 
     file { $dst_dir:
       ensure  => directory,
-      owner   => $datadog_agent::params::dd_user,
+      owner   => $datadog_agent::dd_user,
```

Comment:
I think it is more explicit if they have the same name because params::dd_user is the default value for dd_user variable. Also those variables should never be edited by the user and are only internal variables.